### PR TITLE
Replace write stream with `fs.writeFile` call

### PR DESCRIPTION
**Pull request by rekmarks**
We were creating a write stream in `snaps-cli` and using that to output the bundle, even though we were operating on the entire bundle output by Browserify. This PR removes this useless stream in favor of an `fs.writeFile` call.

### CODE REVIEW

File: packages/snaps-cli/src/cmds/build/utils.ts [hunk 1]
```diff
@@ -42,7 +24,6 @@ type CloseStreamArgs = {
 export async function closeBundleStream({
```
Comment (FrederikBolding) [inline packages/snaps-cli/src/cmds/build/utils.ts:24]: Do we want to rename this?

File: packages/snaps-cli/src/cmds/build/utils.ts [hunk 2]
```diff
@@ -42,7 +24,6 @@ type CloseStreamArgs = {
...
   bundleError,
   bundleBuffer,
-  bundleStream,
   src,
   dest,
   resolve,
```
Comment (FrederikBolding) [inline packages/snaps-cli/src/cmds/build/utils.ts:29]: Arg type `CloseStreamArgs` might also need a rename?

File: packages/snaps-cli/src/cmds/build/utils.ts [hunk 3]
```diff
@@ -53,7 +34,8 @@ export async function closeBundleStream({
   }
 
   try {
```
Comment (FrederikBolding) [inline packages/snaps-cli/src/cmds/build/utils.ts:36]: The error in the try catch might also need a rename?

File: packages/snaps-cli/src/cmds/build/bundle.test.ts [hunk 4]
```diff
@@ -44,7 +41,6 @@ describe('bundle', () => {
         'mockBabelify',
         expect.objectContaining({ global: false }),
       );
-      expect(createStreamMock).toHaveBeenCalledTimes(1);
       expect(closeStreamMock).toHaveBeenCalledTimes(1);
```
Comment (FrederikBolding) [inline packages/snaps-cli/src/cmds/build/bundle.test.ts:44]: Rename all of these if we rename the utility function

**Comment by rekmarks** [comment]
@FrederikBolding there is still a Browserify "bundle" stream that we need to close, and that's what the `closeBundleStream` utility is referring to. This PR removes a second `fs.writeStream` that we were using in addition to the stream created by Browserify.

**Review by FrederikBolding — APPROVED** [review]
lgtm

**Pull request merged**